.ruby-version

```diff
@@ -1 +1 @@
-2.7.2
+2.7.4
```

Comment:
The `parser` gem emits a warning if we're using Ruby 2.7.x but < `2.7.4`. I think it's OK to bump it in Suspenders since we're dealing with minor versions.